internal/sbi/processor/generate_auth_data.go

```diff
@@ -109,7 +109,7 @@ func (p *Processor) ConfirmAuthDataProcedure(c *gin.Context,
 		return
 	}
 
-	c.JSON(http.StatusCreated, createAuthStatusResponse)
+	c.Status(http.StatusCreated)
```

Comment:
![image](https://github.com/user-attachments/assets/258ea2fe-3e54-4f3a-a322-c90606b693ac)

should be`application/json`, and may contain AuthEvent. 
![image](https://github.com/user-attachments/assets/7a7e8a51-709e-435f-a8c4-c38fcfba591f)